build.gradle

```diff
@@ -39,13 +42,12 @@ dependencies {
 	}
 	implementation configurations.localDeps
 
-	compileOnly "com.google.errorprone:javac:9+181-r4173-1"
-
+	errorprone("com.google.errorprone:error_prone_core:2.3.3")
```

Comment:
Isn't `2.11.0` the current version, according to https://mvnrepository.com/artifact/com.google.errorprone/error_prone_core ?

Reply:
Should that also go into `ext.versions`, to put all configurations into one spot?

Reply:
Thanks, fixed!

Reply:
I like option 2: use the newest version and ensure we're running on a compatible runtime.